formatTest/unit_tests/expected_output/wrappingTest.re

```diff
@@ -2216,7 +2216,8 @@ let blah =
 let blahCurriedX x =>
   fun
   /* Comment before first bar */
-  | /* Comment between first bar and OR pattern */ Red x
+  /* Comment between first bar and OR pattern */
+  | Red x
```

Comment:
This is probably the most questionable change, but should be easy to fix if the original is preferred.